specification/columns/skupricedetails.md

```diff
@@ -0,0 +1,54 @@
+# SKU Price Details
+
+The **SKU Price Details** column represents a list of relevant properties shared by all resources having the same [**SKU Price ID**](#skupriceid). These properties enable a practitioner to more easily calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics. They also help a practitioner understand the specifics of a **SKU Price ID**.
+
+
+The _SkuPriceDetails_ column adheres to the following requirements:
+
+* The _SkuPriceDetails_ column MUST be in [_KeyValueFormat_](#key-valueformat).
+* The key for a property SHOULD be formatted in [PascalCase](#glossary:pascalcase).
+* The properties (both keys and values) contained in the _SkuPriceDetails_ column MUST be shared across all charges having the same _SkuPriceId_.
```

Comment:
I feel like this should be a SHOULD. While I don't want to block on this, I'm not sure whether large cloud providers can feasibly do this. This means that, if we want to add a new attribute, then we have to generate a new SkuPriceId. That would impact many teams and how these things are tracked and managed in these large companies. I suspect that this will end up being a "gap" in any provider due to the scale we're working at.

Reply:
This is saying that the set of properties (both keys and values) must apply to all occurrences of a SkuPriceId. Resolved by moving other clauses around to provide clarity.